### PR TITLE
Two finger scrolls

### LINKED ISSUE #97: Two finger scroll broken on mobile

### PR DESCRIPTION
**Pull request by leftynaut**
fixes #97

### CODE REVIEW

File: client/public/assets/css/style.css [hunk 1]
```diff
@@ -34,19 +34,21 @@ nav .brand-logo {
     width: 150px;
 }
 
-#content {
-    position: absolute;
+#canvasWrapper {
```
Comment (shamsup) [inline client/public/assets/css/style.css]: Let's use classes over IDs wherever possible

File: client/public/assets/css/style.css [hunk 2]
```diff
@@ -34,19 +34,21 @@ nav .brand-logo {
...
+    position: relative;
```
Comment (shamsup) [inline client/public/assets/css/style.css]: Does this need to be position relative? What issue does this fix? Why is `34px` the magical offset?

File: client/src/components/whiteboard.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ class Whiteboard extends React.Component {
 
   render() {
     return (
-      <div>
+      <div id="canvasWrapper">
```
Comment (shamsup) [inline client/src/components/whiteboard.js]: Let's prefer classes over IDs. You can't have more than one of the same ID on a page. What if we wanted two whiteboards at the same time?

Comment (leftynaut) [inline client/src/components/whiteboard.js]: Fixed

File: client/public/assets/css/style.css (hunk 1 above)
Comment (leftynaut) [inline client/public/assets/css/style.css]: Fixed

File: client/public/assets/css/style.css (hunk 2 above)
Comment (leftynaut) [inline client/public/assets/css/style.css]: 34px must have been a leftover from earlier code. Fixed.

**Comment by leftynaut** [comment]
Changes have been made. I have two pulls open now, may need to rebase once one is merged.

**Pull request merged**